rgb_follower.py

```diff
@@ -77,11 +77,11 @@ async def main():
     print("connected")
     camera = Camera.from_robot(robot, "my_camera")
     base = Base.from_robot(robot, "scuttlebase")
-    vision = VisionClient.from_robot(robot, "builtin")
+
     # Put your detector name in place of "green_detector"
-    detections = await vision.get_detections_from_camera("my_camera", "green_detector")
-    names = await vision.get_detector_names()
-    print(names)
+    green_detector = VisionClient.from_robot(robot, "green_detector")
+    detections = await green_detector.get_detections_from_camera(camera)
+    print(detections)
```

Comment:
What if there are no detections? My original intent was to confirm that the code successfully connected and got a detector and camera from the robot, not to print detections (since there might not be any yet, and because printing just one set of detections wouldn't be too useful anyway). Do you have a different intent with this change?

Reply:
That's fine with me we can remove these two lines?

Reply:
Yeah, or leave as-was?

Reply:
I thought that didn't work anymore but might be wrong